game.js

```diff
@@ -0,0 +1,62 @@
+function drawmap(){
+    var canvas = document.getElementById("main");
+    var context = canvas.getContext("2d");
+
+    for (var i=0; i<3; i++){
```

Comment:
what if I want to play in a board with 4x4 - you need to rewrite your code? (optional to fix)